RealmSwift/RealmCollection.swift

```diff
@@ -123,6 +122,11 @@ private func forceCast<A, U>(_ from: A, to type: U.Type) -> U {
     return from as! U
 }
 
+/// A type which can be stored in a Realm List or Results
+public protocol RealmCollectionValue {
+    static func className() -> String
```

Comment:
This function has actually gone away on the primitive arrays branch because different functions were needed.